tests/security/cc/cc_selinux_setup.pm

```diff
@@ -0,0 +1,40 @@
+# Copyright (C) 2021 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+#
+# Summary: Setup "audit-test" test environment of a system running SELinux
+# Maintainer: llzhao <llzhao@suse.com>
+# Tags: poo#93441
+
+use base 'selinuxtest';
+use strict;
+use warnings;
+use testapi;
+use utils;
+
+sub run {
+    my ($self) = @_;
+    my $selinux_config_file = "/etc/selinux/config";
+    select_console "root-console";
+
+    my $file = "selinux-policy-targeted-20210419-5.2.noarch.rpm";
+    assert_script_run "wget --quiet " . data_url("selinux/$file");
+    assert_script_run("rpm -ivh --nosignature --nodeps --noplugins $file");
+
+    # Set SELINUXTYPE=targeted
+    # TODO: "targeted" type still has some problems using "minimum" atm for a workaround
```

Comment:
@lilyeyes 
Can you please add some more detail information here why "targeted" can't be used? any bugs or poos opened?

Reply:
> @lilyeyes
> Can you please add some more detail information here why "targeted" can't be used? any bugs or poos opened?

Following the docs cert team offered. no bug opened atm as it is still under investigation (time consuming also). I used a workaround in order to not block other test cases integration.